fetool/pages/clients.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import useSWR from "swr";
+import { Box } from "@mui/system";
+import axios from "axios";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import apiList from "../apiRoutes/apiNames";
+
+const fetcher = async () => {
+  const response = await axios.get(
+    `${process.env.NEXT_PUBLIC_API_URL}/${apiList[0]}?fields=name&populate=delivery_head%2Cpoint_of_contacts%2Cprojects`
+  );
+  const clientData = response.data;
+  return clientData;
+};
+
+const Client = () => {
+  const { data, error } = useSWR("clientDetails", fetcher); //(uniquekey,fetcher function)

```

Comment:
The unique key can be the endpoint and sent as a parameter to fetcher function. Check the documentation and make the changes accordingly.

Reply:
Changed it